github-collaboration/github-collaboration.md

```diff
@@ -44,15 +45,20 @@ In the industry, apps are created by teams of developers rather than individual
 Instructor demonstrates in the video walk through how to work with work with pull requests on GitHub
 
 ### Independent Practice  
-- TODO, but rough sketch:
-    - Create a GitHub repo with a few files in it
-    - Apprentices clone the repo and modify one of the files
-    - Create branch, commit, create PR
-    - Instructor modifies base branch to cause a conflict
-    - Apprentices resolve merge conflicts and merge their PRs
+- Create a repository on GitHub
```

Comment:
This is great, Himanshu!  Could you please make the changes we discussed on Thursday?  They were:
1. Re-organize tasks into guided practice with a shared screen v independent practice on the apprentice's own machine.
2. Expand on each direction, making the task very clear to new students and teachers, ie: Here is a link to the practice repo: "ajkdfj".  Press the fork button.  Enter these commands in your terminal to clone the repo and open it in VS Code. Add name to line 1 of practice.md...."